frontend/src/components/PresetList.tsx

```diff
@@ -0,0 +1,76 @@
+"use client"
+
+import React, { ReactNode } from "react"
+
+import Link from "next/link"
+
+import classNames from "classnames"
+
+import AsyncButton from "@/components/AsyncButton"
+import Button from "@/components/Button"
+import LoadingSpinner from "@/components/loading.svg"
+import { PlatformIcon } from "@/components/PlatformSelect/PlatformIcon"
+import styles from "@/components/ScratchList.module.scss"
+import * as api from "@/lib/api"
```

Comment:
Ideally we don't introduce new code that does this, use more granular imports

Reply:
I change to granular import instead of wildcard